### PR TITLE
Added version support info

### PR DESCRIPTION
**Pull request by prmadhes-msft**
just a small update to add the MI backup is supported with Enterprise, Developer, or Standard SQL Server version at On-Prem.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@MladjoA 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/managed-instance/restore-database-to-sql-server.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ This article describes how to restore a database backup from Azure SQL Managed I
 
 ## Overview
 
-The database format alignment between SQL Managed Instance and SQL Server 2022 gives you an easy way to copy or move databases from your managed instance to a SQL Server 2022 instance that's hosted on-premises, on virtual machines in Azure, or in other clouds. 
+The database format alignment between SQL Managed Instance and SQL Server 2022 gives you an easy way to copy or move databases from your managed instance to a SQL Server 2022 instance that's hosted on-premises of SQL Server version Enterprise, Developer or Standard, on virtual machines in Azure, or in other clouds.
```
Comment (MladjoA) [inline azure-sql/managed-instance/restore-database-to-sql-server.md]: @Jak-MS , I'd move list of supported editions to another place within the sentence, like this:

...gives you an easy way to copy or move databases from your managed instance to SQL Server 2022 instance with Enterprise, Developer, or Standard edition installed, that's hosted....

Comment (prmadhes-msft) [reply azure-sql/managed-instance/restore-database-to-sql-server.md]: Yes, it make sense.

**Comment by Court72** [comment]
@prmadhes-msft - Can you make @MladjoA's suggested change?


@MladjoA - When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

Thanks!

File: azure-sql/managed-instance/restore-database-to-sql-server.md (hunk 1 above)
Comment (prmadhes-msft) [reply azure-sql/managed-instance/restore-database-to-sql-server.md]: rephrase sentence as advised

**Comment by MladjoA** [comment]
#sign-off

**Pull request merged**